exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return arr.length === 0; // complete this statement
```

Comment:
Hi Petar, I have a question for you. :sweat_smile: 
Do you think === (triple equals) is right to use this statement?